cider/tests/functionality/CiderAggArrowTest.cpp

```diff
@@ -1019,27 +1019,53 @@ TEST_F(CiderPartialAVGFpArrowTest, mixedGroupbyPartialAVG) {
   std::vector<int64_t> expect_count3({0, 1, 1});
   std::vector<int64_t> expect_count4({0, 0, 0});
 
-  ArrowSchema* expect_schema;
-  ArrowArray* expect_array;
+  ArrowSchema** expect_schemas = (ArrowSchema**)malloc(4 * sizeof(ArrowSchema*));
+  ArrowArray** expect_arrays = (ArrowArray**)malloc(4 * sizeof(ArrowArray*));
```

Comment:
maybe this two variable cause memory leak?
emm, this is tests codes, maybe not consider too much...